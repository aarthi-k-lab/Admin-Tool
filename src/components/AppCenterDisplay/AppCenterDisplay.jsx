import React from 'react';
import PropTypes from 'prop-types';
import AppContainer from '../AppContainer';
import Body from '../Body';
import Center from '../Center';

import './AppCenterDisplay.css';

const AppCenterDisplay = ({ header, children }) => (
  <AppContainer>
    { header }
    <Body>
      <Center>
        <div styleName="app-center-display">
          {children}
        </div>
      </Center>
    </Body>
  </AppContainer>
);

AppCenterDisplay.defaultProps = {
  header: null,
};

AppCenterDisplay.propTypes = {
  children: PropTypes.node.isRequired,
  header: PropTypes.node,
};

export default AppCenterDisplay;
