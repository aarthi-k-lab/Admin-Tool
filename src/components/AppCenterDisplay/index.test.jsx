import React from 'react';
import renderer from 'react-test-renderer';
import AppCenterDisplay from '.';

describe('<AppCenterDisplay />', () => {
  test('snapshot test', () => {
    const tree = renderer
      .create(<AppCenterDisplay header="header">content</AppCenterDisplay>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
