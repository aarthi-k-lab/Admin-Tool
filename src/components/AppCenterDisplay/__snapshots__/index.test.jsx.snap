// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AppCenterDisplay /> snapshot test 1`] = `
<div
  className="components-AppContainer-AppContainer__app--_pvE9"
>
  header
  <div
    className="components-Body-Body__body-container--3Iujq"
  >
    <section
      className="components-Body-Body__body--3PVBY"
    >
      <div
        className="components-Center-Center__center--expand--1OMMm components-Center-Center__center--QKyYo"
      >
        <div
          className="components-AppCenterDisplay-AppCenterDisplay__app-center-display--2ZCSr"
        >
          content
        </div>
      </div>
    </section>
  </div>
  <footer
    className="components-Footer-Footer__footer--2oWe5"
  >
    Â© Nationstar Mortgage LLC, All rights reserved
  </footer>
</div>
`;
